pkg/state/state_test.go

```diff
@@ -3458,14 +3458,11 @@ func TestCommonDiffFlags(t *testing.T) {
 			stripTrailingCR: true,
 			expected: []string{
 				"--strip-trailing-cr",
-				"--reset-values",
```

Comment:
why change here?

Reply:
This is a test for "commonDiffFlags", since with this change the values control flags (reset/reuse values) depend on the release, I moved them out of the "common" flags, so I also needed to modify the test to not expect these flags anymore